plugins/colorize/colorize.plugin.zsh

```diff
@@ -46,7 +46,11 @@ colorize_cat() {
         if [[ "$ZSH_COLORIZE_TOOL" == "pygmentize" ]]; then
             pygmentize -O style="$ZSH_COLORIZE_STYLE" -g
         else
-            chroma --style="$ZSH_COLORIZE_STYLE"
+            if [ -z "$ZSH_COLORIZE_CHROMA_FORMATTER" ]; then
+                chroma --style="$ZSH_COLORIZE_STYLE"
+            else
+                chroma --style="$ZSH_COLORIZE_STYLE" --formatter="$ZSH_COLORIZE_CHROMA_FORMATTER"
+            fi
```

Comment:
[The default `--formatter` is `terminal`](https://github.com/alecthomas/chroma/blob/477ad4aefdf2501f92e0a2261572bfc2cc2c9cb2/cmd/chroma/main.go#L49). We can use that to simplify the code by always providing a `--formatter` argument:
```suggestion
            chroma --style="$ZSH_COLORIZE_STYLE" --formatter="${ZSH_COLORIZE_CHROMA_FORMATTER:-terminal}"
```
Same as for the line below.